scalecube-organization/src/main/java/io/scalecube/organization/operation/AddOrganizationApiKey.java

```diff
@@ -12,15 +12,20 @@
 import io.scalecube.organization.tokens.TokenVerifier;
 import io.scalecube.organization.tokens.store.ApiKeyBuilder;
 import io.scalecube.organization.tokens.store.KeyStore;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
 import java.util.EnumSet;
+import java.util.UUID;
 
 public class AddOrganizationApiKey
     extends ServiceOperation<AddOrganizationApiKeyRequest, GetOrganizationResponse> {
 
+  private final KeyPairGenerator keyPairGenerator;
   private final KeyStore keyStore;
 
   private AddOrganizationApiKey(Builder builder) {
     super(builder.tokenVerifier, builder.repository);
```

Comment:
why not just to pass builder as reference?

Reply:
Changed. Done.